sonic-utilities-tests/sku_create_test.py

```diff
@@ -51,30 +69,197 @@ def are_file_contents_same(self,fname1,fname2):
         f2.close()
         return True
     
-    def test_no_param(self):
-        if (os.path.exists(output_dir_path)):
-            shutil.rmtree(output_dir_path)
+    def test_sku_from_xml_file(self):
+        if (os.path.exists(output_xml_dir_path)):
+            shutil.rmtree(output_xml_dir_path)
 
-        my_command = sku_create_script + " -f "  + sku_def_file  + " -d " + input_path
+        my_command = sku_create_script + " -f "  + sku_def_file  + " -d " + xml_input_path
 
         #Test case execution without stdout
         result = subprocess.check_output(my_command,stderr=subprocess.STDOUT,shell=True)
         print result
 
         #Check if the Output file exists
-        if (os.path.exists(output_file_path)):
-            print("Output file: ",output_file_path,"exists. SUCCESS!")
+        if (os.path.exists(output_xml_file_path)):
+            print("Output file: ",output_xml_file_path, "exists. SUCCESS!")
```

Comment:
Add space after commas in parameter lists. Do this throughout the file.

TIP: You can pass your Python code through `autopep8`. I suggest the following: `autopep8 --max-line-length 120 <file>`

Reply:
addressed in next commit